test/e2e/stackedetcd_test.go

```diff
@@ -0,0 +1,53 @@
+// +build e2e
+
+package e2e
+
+import (
+	"testing"
+
+	"github.com/aws/eks-anywhere/internal/pkg/api"
+	"github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+	"github.com/aws/eks-anywhere/test/framework"
+)
+
+func runStackedEtcdFlow(test *framework.E2ETest) {
+	test.GenerateClusterConfig()
+	test.CreateCluster()
+	test.DeleteCluster()
+}
+
+func TestVSphereKubernetes120StackedEtcdBottlerocket(t *testing.T) {
+	test := framework.NewE2ETest(t,
+		framework.NewVSphere(t, framework.WithBottleRocket120()),
+		framework.WithClusterFiller(api.WithKubernetesVersion(v1alpha1.Kube120)),
+		framework.WithClusterFiller(api.WithControlPlaneCount(1)),
```

Comment:
I think let's keep the cp count 3 here and for test `TestVSphereKubernetes120StackedEtcdUbuntu` too and we can remove test `TestVSphereKubernetes120StackedEtcdBottlerocket3CP2Workers` since it'll get covered by this test